src/_Configuration/Framework.Configuration.Generated.DAL.NHibernate/Mapping/LogMessageMapping.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Framework.Configuration.Domain;
+using Framework.Configuration.Generated.DAL.NHibernate.Mapping.Base;
+
+namespace Framework.Configuration.Generated.DAL.NHibernate.Mapping
+{
+    public class LogMessageMapping : CfgBaseMap<LogMessage>
+    {
+        public LogMessageMapping()
+        {
+            this.Map(x => x.Action).Length(512);
+            this.Map(x => x.InputMessage).Length(int.MaxValue);
+            this.Map(x => x.OutputMessage).Length(int.MaxValue);
+            this.Map(x => x.UserName).Length(512);
+            this.Component(
```

Comment:
You can define Period mapping once by [ComponentMap](https://github.com/nhibernate/fluent-nhibernate/wiki/Fluent-mapping#componentmap)